vowpalwabbit/model_merger/src/main.cc

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
```

Comment:
can the merge functions be called not via the merger tool but during normal vw execution by mistake? any reduction could potentially call merge should add a warning comment or something?

Reply:
A reduction is unlikely to have the arguments available to be able to call it